libs/community/langchain_community/embeddings/nemo.py

```diff
@@ -40,14 +41,22 @@ def is_endpoint_live(url: str, headers: Optional[dict], payload: Any) -> bool:
         # Handle any exceptions (e.g., connection errors)
         raise Exception(f"Error querying the endpoint: {e}")
 
-
 class NeMoEmbeddings(BaseModel, Embeddings):
```

Comment:
could we use the @deprecated decorator instead? see [example](https://github.com/langchain-ai/langchain/blob/64e17bd793483b63092407cf79d434179436b250/libs/community/langchain_community/chat_models/openai.py#L147)